Source/WorkersES6/decodeDraco.js

```diff
@@ -317,6 +337,32 @@ function decodePrimitive(parameters) {
         dracoDecoder,
         dracoAttribute
       );
+      if (parameters.dequantizeInShader && attributeName !== "NORMAL") {
```

Comment:
@ebogo1 correction, from the [B3DM Spec](https://github.com/CesiumGS/3d-tiles/tree/main/specification/TileFormats/Batched3DModel#binary-gltf) it's called `_BATCHID` (all one word). It's probably due to the fact that this is not `NORMAL`, but it's not one of the built-ins like `POSITION` either.

I wonder if it's going down the wrong path due to this if statement.

Reply:
Good catch - with the code in `main` it looks like we wouldn't expect to use Draco's quantization transform on anything other than the properties in `var attributesToSkip = ["POSITION", "NORMAL", "COLOR", "TEX_COORD"];`.